app/helpers/ukims_helper.rb

```diff
@@ -0,0 +1,25 @@
+module UkimsHelper
+  CUT_OFF_DATE = Date.new(2023, 9, 30)
+
+  def uk_internal_market_scheme
+    if cut_off_validation
+      'UK Internal Market Scheme'
+    else
+      'UK Trader Scheme'
+    end
+  end
+
+  def ukims_annual_turnover
```

Comment:
with _ looks better:
`uk_ims_annual_turnover`

Reply:
hmmm i kept it as all together as 'u-kims' is the way we are pronouncing uk-ims is confusing me ha..